e2e/src/cli/specs/upload.e2e-spec.ts

```diff
@@ -1,9 +1,63 @@
 import { LoginResponseDto, getAllAlbums, getAssetStatistics } from '@immich/sdk';
-import { readFileSync } from 'node:fs';
+import { cpSync, readFileSync } from 'node:fs';
 import { mkdir, readdir, rm, symlink } from 'node:fs/promises';
-import { asKeyAuth, immichCli, testAssetDir, utils } from 'src/utils';
+import { asKeyAuth, immichCli, specialCharStrings, testAssetDir, utils } from 'src/utils';
 import { beforeAll, beforeEach, describe, expect, it } from 'vitest';
 
+interface Test {
+  test: string;
+  paths: string[];
+  files: Record<string, boolean>;
+}
+
+const tests: Test[] = [
+  {
+    test: 'should support globbing with *',
+    paths: [`/photos*`],
+    files: {
+      '/photos1/image1.jpg': true,
+      '/photos2/image2.jpg': true,
+      '/images/image3.jpg': false,
+    },
+  },
```

Comment:
Can you add a test like this but you just give it a path to a folder with a space? Something like `upload -r ./My Photos/2002` and it works?

Reply:
Done! I also added a few more cases I overlooked